src/openlifu/bf/sequence.py

```diff
@@ -38,18 +40,12 @@ def from_dict(d):
         :param d: Dictionary of the sequence parameters
         :returns: Sequence object
         """
-        return Sequence(pulse_interval=d["pulse_interval"],
-                        pulse_count=d["pulse_count"],
-                        pulse_train_interval=d["pulse_train_interval"],
-                        pulse_train_count=d["pulse_train_count"])
+        return Sequence(**d)
 
     def to_dict(self):
```

Comment:
This function member seems to repeat itself in many other dataclasses. Better to create a mother class that implements them (`to_dict` and `from_dict`).